Scripting/Bindings.cs

```diff
@@ -236,9 +429,10 @@ public override void _Process(float Delta)
 			return;
 		}
 
+		bool KeyPressed = false; // This is to make sure console controls don't interfere with K&M
```

Comment:
This could easily cause issues with getting input states which do not match the actual inputs from kb, mouse, or controller. Instead the BindingObject could have a bool indicating if the joystick was in the deadzone last check and only fire the event if `!InDeadzone || (InDeadzone && !OldInDeadzone)` (I think this is correct but you will likely have to fiddle with this)